.travis.yml

```diff
@@ -66,14 +66,15 @@ install:
   # CMAKE                                                                     #
   #############################################################################
   - export PATH=$CMAKE_ROOT/bin:$PATH
-  - CMAKE_33_FOUND=$(cmake --version | grep " 3\.3\." >/dev/null && { echo 0; } || { echo 1; })
-  - if [ $CMAKE_33_FOUND -ne 0 ]; then
+  - CMAKE_39_FOUND=$(cmake --version | grep " 3\.9\." >/dev/null && { echo 0; } || { echo 1; })
```

Comment:
@psychocoderHPC wouldn't it be safer to test with CMake 3.7.0 here to be sure one is not accidentally including newer features in upcoming PRs?